org.civicrm.reqangular/src/common/angular-date/src/decorators/DatepickerDirectiveDecorator.js

```diff
@@ -0,0 +1,24 @@
+function DatepickerDirectiveDecorator($delegate) {
+    var directive = $delegate[0];
+
+    var old_link = directive.link;
+
+    delete directive.link;
```

Comment:
Is this necessary? I'm not really familiar with decorators to be honest, but i was reading this [article here](http://angular-tips.com/blog/2013/09/experiment-decorating-directives/) and it does seem there is a necessity to delete the link function